Add release notes for R25 (#583)
brightlayer-ui/doc-it: src/docs/release-notes/R25/Summary.mdx

> Migrated all our [Angular](https://brightlayer-ui-components.github.io/angular) and [React](https://brightlayer-ui-components.github.io/react) component developer documentation from Storybook to a new site with more examples and more detailed usage explanations; added support for React Native v0.69; added [icon style guide](/style/icon-guidelines).

Reviewer: See my first comment, I would restructure this sentence just a little bit to put more emphasis on what's new vs storybook.

> Migrated all our [Angular](https://brightlayer-ui-components.github.io/angular) and [React](https://brightlayer-ui-components.github.io/react) component developer documentation from Storybook to a new site with powerful features such as Examples page that provide usage context and Playground pages where you can customize components and generate code snippets; added support for React Native v0.70; added [icon style guide](/style/icon-guidelines).

Reviewer 2: Since this is the summary, it doesn't have to be as detailed:
```suggestion
Migrated all our [Angular](https://brightlayer-ui-components.github.io/angular) and [React](https://brightlayer-ui-components.github.io/react) component documentation from Storybook to an in-house site with powerful new features, added support for React Native v0.70.x, and added an [icon style guide](/style/icon-guidelines).
```
